packages/loader/src/index.ts

```diff
@@ -14,11 +15,13 @@ import "./SpriteAtlasLoader";
 import "./SpriteLoader";
 import "./Texture2DLoader";
 import "./TextureCubeLoader";
-import "./AnimationClipLoader";
+import "./ktx2/KTX2Loader";
 
-export { parseSingleKTX } from "./compressed-texture";
 export type { GLTFParams } from "./GLTFLoader";
-export * from "./resource-deserialize";
 export * from "./SceneLoader";
 export type { Texture2DParams } from "./Texture2DLoader";
+export { parseSingleKTX } from "./compressed-texture";
 export * from "./gltf";
+export { KTX2Loader } from "./ktx2/KTX2Loader";
+export { KTX2TargetFormat } from "./ktx2/KTX2TargetFormat";
```

Comment:
Should export to user like `Texture2DParams`

Reply:
done